helm/cas-ggircs/templates/app/cronJobs/cron-deploy-data.yaml

```diff
@@ -70,3 +70,4 @@ spec:
                 - |
                   set -euo pipefail;
                   cd schema && sqitch deploy;
+                  ./test/dev/deploy_dev_data.sh --oc-project=$(NAMESPACE);
```

Comment:
@pbastia @matthieu-foucault I'm unsure about whether this command is correct.

Reply:
I see 
```bash 
if [[ $1 =~ -dev$ ]]; then ...
``` 
in the script, so I think we might have to give it a regex `.*-dev$`

I think this makes the script callable with anything that ends with dev, not necessarily `--oc-project=...` ?

Reply:
I'd rather not have the `--oc-project` param, and conditionally add the `./test/dev/deploy_dev_data.sh` call to the job template when the namespace suffix is `dev`. The script is named `deploy_dev_data`, it should only be called in a dev environment

Reply:
This also has to be done at the end of the etl job, as it assumes that the swrs schema was loaded. The fuel_mapping table is empty otherwise